### PR TITLE
xbutil2 enhancements

### PR DESCRIPTION
**Pull request by AShivangi**
- CR-1077398 xbmgmt2 testing - clarify the note for xbmgmt --new program revert-to-golden
- XRT-768 Improve the wording of the --revert-to-golden description
- CR-1070769 Align time zone convention in XRT output
- XRT-740 Rename xbutil --report "fan" to "mechnanical" and insert the fan as a sub-section

```
$ xbutil -new examine -r mechanical -d 65:00.1
-----------------------------------------------------
 Invoking next generation of the xbutil application
-----------------------------------------------------
Mechanical
  Fans
    FPGA Fan 1
      Critical Trigger Temp : 31 C
      Speed                 : 0 RPM
```

- CR-1077429 Improve messaging for BDF specified incorrect or not found
- CR-1076968 xbutil --new examine --r verbose does not make it clear which output belongs to which card
- CR-1077377 xbutil 2 testing feedback on xbutil --new examine -r vebose **(fixed except aync errors and debug-ip-layout)**

Fixed:
- Catch error and only print available devices when a wrong bdf is entered 
- Update thresholds for bw testcases

### CODE REVIEW

File: src/runtime_src/core/tools/common/ReportCu.cpp [hunk 1]
```diff
@@ -63,12 +63,11 @@ populate_cus(const xrt_core::device *device)
     ip_buf = xrt_core::device_query<qr::ip_layout_raw>(device);
     cu_stats = xrt_core::device_query<qr::kds_cu_info>(device);
   } catch (const std::exception& ex){
-    pt.put("ERROR: ", ex.what());
```
Comment (rozumx) [inline src/runtime_src/core/tools/common/ReportCu.cpp:66]: Great catch

File: tests/python/23_bandwidth/23_bandwidth.py [hunk 2]
```diff
@@ -52,7 +52,7 @@ def getThreshold(devHandle):
         threshold = 20000
     if b"u2x4" in deviceInfo.mName or b"U2x4" in deviceInfo.mName or b"u2_gen3x4" in deviceInfo.mName:
         threshold = 10000
-    if b"_u25_" in deviceInfo.mName: # so that it doesn't set theshold for u250
+    if b"_u25_" in deviceInfo.mName or b"_u30_" in deviceInfo.mName: # so that it doesn't set theshold for u250
```
Comment (rozumx) [inline tests/python/23_bandwidth/23_bandwidth.py:55]: I know why this code is here.  Please please please, post 2020.2 lets find a better way of doing this.

**Review by rozumx — APPROVED** [review]

File: tests/python/23_bandwidth/23_bandwidth.py (hunk 2 above)
Comment (AShivangi) [reply tests/python/23_bandwidth/23_bandwidth.py:55]: Yes. This is the reason why we are introducing platform.json, to make it more data driven

File: src/runtime_src/core/common/time.cpp [hunk 3]
```diff
@@ -18,8 +18,6 @@
 #include "time.h"
 
 #include <chrono>
-#include <string>
-#include <ctime>
```
Comment (stsoe) [inline src/runtime_src/core/common/time.cpp:22]: I am reverting this change in #4259.

File: src/runtime_src/core/common/time.h [hunk 4]
```diff
@@ -19,6 +19,7 @@
 
 #include "core/common/config.h"
 #include <string>
+#include <ctime>
```
Comment (stsoe) [inline src/runtime_src/core/common/time.h:22]: Please don't include unnecessary header files.  You should include exactly what you need in order to compile the header file stand-alone and <ctime> is not needed.  Reverted in #4259

**Review by stsoe — COMMENTED** [review]
Minor comments.

**Pull request merged**